compact/definitions.xml

```diff
@@ -606,7 +605,7 @@ Service gaps in FW direction (before endcapP ECAL) and BW direction (before endc
     <constant name="HcalEndcapN_length"   value="44.0*cm"/>
     <constant name="HcalEndcapN_zmax"     value="HcalEndcapN_zmin + HcalEndcapN_length"/>
     <comment> HcalEndcapN needs to clear the straight beampipe with 1 * crossing angle + its radius of 22.25 mm </comment>
-    <constant name="HcalEndcapN_rmin"     value="max((HcalEndcapN_zmin + HcalEndcapN_length) * tan(abs(CrossingAngle)) + 22.25 * mm, 11 * cm)"/>
+    <constant name="HcalEndcapN_rmin"     value="19.9431*cm"/>
```

Comment:
These are not hard-coded values that should be in the definitions file. These are implementation details only relevant to the HcalEndcapN. The definitions file should be like the detector parameter table. This is not a number in the detector parameter table, so it should not be in the definitions file.

Same with rmax.